machine_learning/classifier.py

```diff
@@ -0,0 +1,127 @@
+import sys
+import pandas as pd
+import numpy as np
+from sklearn.metrics import precision_recall_fscore_support
+from sklearn.metrics import accuracy_score
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.datasets import make_classification
+
+from gensim.models.keyedvectors import KeyedVectors
+model = KeyedVectors.load_word2vec_format("glove_model2.txt", binary=False, limit=50000)
+
+
+def feature_extractor(orig_object, data_object):
+    """
+    This function is used to extract features in the data set and return a list of all the features computed.
+    :param orig_object:
+    :param data_object:
+    :return feat(list):
```

Comment:
would be good to specify a brief summary of the objects.

Reply:
added them, thanks !